otx/cli/builder/builder.py

```diff
@@ -208,10 +208,11 @@ def build_task_config(
         template_config.dump(str(workspace_path / "template.yaml"))
 
         # Create Data.yaml
-        data_subset_format = {"ann-files": None, "data-roots": None}
-        data_config = {"data": {subset: data_subset_format.copy() for subset in ("train", "val", "test")}}
-        data_config["data"]["unlabeled"] = {"file-list": None, "data-roots": None}
-        mmcv.dump(data_config, str(workspace_path / "data.yaml"))
+        if (workspace_path / "data.yaml").exists() is False:
+            data_subset_format = {"ann-files": None, "data-roots": None}
```

Comment:
I think this code part is exactly same as `ConfigManager._create_empty_data_cfg()`. How about making it static function and reusing it?